### PR TITLE
Livestream handling with VLCj

### PR DESCRIPTION
**Pull request by tkolenbrander**
Added the LiveStreamHandlers class to handle the cameralivestream using the VLCj library. The livestream is currently only implemented in the preset view. This fixes issue #16

### CODE REVIEW

File: src/main/java/nl/tudelft/contextproject/gui/LiveStreamHandler.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import uk.co.caprica.vlcj.player.direct.DirectMediaPlayer;
+import uk.co.caprica.vlcj.player.direct.format.RV32BufferFormat;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Handler for streaming media into the GUI through VLC.
+ * 
+ * @since 0.5
+ */
```
Comment (stevenmeijer) [inline src/main/java/nl/tudelft/contextproject/gui/LiveStreamHandler.java:28]: `Version should be 0.6 :)

**Comment by stevenmeijer** [comment]
Looks good to me, great work!

File: src/main/java/nl/tudelft/contextproject/gui/PresetController.java [hunk 2]
```diff
@@ -52,7 +51,9 @@
     @FXML private TableColumn<Preset, String> descColumn;
     @FXML private VBox vBox;
 
+    private LiveStreamHandler streamHandle;
     private ObservableList<Preset> data = FXCollections.observableArrayList();
+    private ImageView imageView;
```
Comment (TimYuenior) [inline src/main/java/nl/tudelft/contextproject/gui/PresetController.java:56]: Just some small subjective input: I personally prefer "streamHandler" over "streamHandle".

**Comment by TimYuenior** [comment]
Seems good, works as far as I see. Only put a small suggestion above. Can be merged as far as I'm concerned

**Comment by tkolenbrander** [comment]
@TimYuenior Fixed it!

**Comment by TimYuenior** [comment]
@tkolenbrander Awesome. WIll merge it now.

**Pull request merged**